src/test/java/seedu/address/testutil/StudentBuilder.java

```diff
@@ -14,9 +20,9 @@
     public static final String DEFAULT_NAME = "Amy Bee";
     public static final String DEFAULT_PHONE = "85355255";
     public static final String DEFAULT_EMAIL = "amy@gmail.com";
-    public static final String DEFAULT_ADDRESS = "123, Jurong West Ave 6, #08-111";
-    public static final String DEFAULT_STUDY_LEVEL = "Sec 5";
-    public static final String DEFAULT_GUARDIAN_PHONE = "33333333";
+    public static final String DEFAULT_ADDRESS = "Blk 136 Jurong West Ave 6, #08-111";
+    public static final String DEFAULT_STUDY_LEVEL = "Secondary 5";
+    public static final String DEFAULT_GUARDIAN_PHONE = "91234567";
```

Comment:
Hmmmm honestly I tot that changing the values for amy bee here will will affect places like the CommandTestUtil.java as well and fails the test case. But seems like it passes the test case. Seems like they are not linked then. But just in case next time there is some error for this, we might need to check back ah.

Reply:
I will check again if there are conflicts with test cases. But the reason for the change was to fix a formatting for the study level field because some of the entries differ, eg. "P5", "Sec 2", "JC 1.